blog/2024-11-27-release-comms-data-link/index.mdx

```diff
@@ -0,0 +1,58 @@
+---
+slug: comms-nov2024
+title: |
+  🎉 Highlights from the Latest RenkuLab Releases! 🚀
+authors: [laura]
+tags:
+  - Release
+  - Renku 2.0
+  - Data Connector
+image: ./image_1.png
+description: Reuse data connectors across projects and bring your own image into Renku
+---
+
+![Renku Data Connectors](./image_1.png)
+
+We’ve been busy behind the scenes, and now it’s time to share some of the exciting updates we’ve rolled out over our last few releases. These features are part of the Renku 2.0 beta, designed to to empower collaboration and make it easier to connect and share data, code, and compute resources. Here’s a quick tour of what’s new.
+
+<!-- truncate -->
+
+### **Reuse Data Connectors Across Projects**
+
+Sharing is caring, and now Renku makes it easier than ever to share and reuse data connectors (formerly called data sources) across projects. Instead of re-entering connection details every time, you can now reuse existing data connectors from your group or even other shared projects!
+
+📌 *How it works*: Copy the identifier of an existing data (1) and link it to your project with just a few clicks (2,3).
+
+![Copy and reuse a data connector](./image_2.png)
+
+This feature saves time, reduces duplication, and encourages collaboration. We can’t wait to see how you’ll use it!
+
+---
+
+### **Expanded Support to “Bring Your Own Image” to RenkuLab**
+
+Calling all power users: You now have more flexibility when running custom images in RenkuLab sessions! Previously, your image needed to include JupyterServer. That’s no longer the case! Now, you can bring almost any Docker image that fits your needs.
+
+🔗 Check out the How To guide: [How to use your own docker image for a session launcher](https://www.notion.so/How-to-use-your-own-docker-image-for-a-session-launcher-11f0df2efafc80af848ffcaf9ccff31c?pvs=21)
```

Comment:
I accidentally put the notion app link, rather than the weblink!
```suggestion
🔗 Check out the How To guide: [How to use your own docker image for a session launcher](https://renku.notion.site/How-to-use-your-own-docker-image-for-a-session-launcher-11f0df2efafc80af848ffcaf9ccff31c)
```